validator/remote/src/main/java/tech/pegasys/teku/validator/remote/EventTypes.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.remote;
+
+public class EventTypes {
+  public static final String HEAD = "head";
+  public static final String CHAIN_REORG = "chain_reorg";
+}
```

Comment:
this is basically `tech.pegasys.teku.beaconrestapi.handlers.v1.events.EventType` - should we use the same?

Reply:
It wasn't in an accessible module before (was in with all the implementation of the REST API).  I've moved it to be with the other API response schema stuff and now use it.